src/mono/wasm/build/WasmApp.Native.targets

```diff
@@ -185,8 +185,8 @@
       <_EmccCompileBitcodeRsp>$(_WasmIntermediateOutputPath)emcc-compile-bc.rsp</_EmccCompileBitcodeRsp>
       <_EmccLinkRsp>$(_WasmIntermediateOutputPath)emcc-link.rsp</_EmccLinkRsp>
 
-      <EmccInitialMemory Condition="'$(EmccInitialMemory)' == ''">$(EmccTotalMemory)</EmccInitialMemory>
-      <EmccInitialMemory Condition="'$(EmccInitialMemory)' == ''">536870912</EmccInitialMemory>
+      <EmccInitialHeapSize Condition="'$(EmccInitialHeapSize)' == ''">$(EmccTotalMemory)</EmccInitialHeapSize>
+      <EmccInitialHeapSize Condition="'$(EmccInitialHeapSize)' == ''">536870912</EmccInitialHeapSize>
```

Comment:
We should evaluate whether ~536mb is actually a reasonable initial heap size. It may be too big and causing OOM for applications that won't ever use that much